main.js

```diff
@@ -8,6 +8,8 @@ var express = require('express');
 var errorsHandler = require('express-errors-handler');
 var flags = require('next-feature-flags-client');
 var expressHandlebars = require('express-handlebars');
+var barriers = require('next-barrier-component')(flags);
```

Comment:
it seems to be from looking at the code, but I'm not going to use it anyway, so I'll take it out...